README.md

```diff
@@ -1,488 +1,51 @@
-```
-
-### Protobuf
-```
-# Sanity check
-if [ -n "`which protoc`" ]; then   
-    echo -e "\\e[33mWarning: protoc is already installed in this system. Proceed with caution.\\e[0m"; 
-fi
-
-curl -OL https://github.com/google/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
-unzip protoc-3.12.3-linux-x86_64.zip -d protoc3
-mv ./protoc3/*/ /usr/local/bin/
-chown root:root /usr/local/bin/protoc
-chown -R root:root /usr/local/include/google
-
-# Install protoc-gen-go
-go get -u google.golang.org/protobuf/cmd/protoc-gen-go
-go install google.golang.org/protobuf/cmd/protoc-gen-go
-```
-
-### Ign-transport
-
-And then
-```
-echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" > /etc/apt/sources.list.d/gazebo-stable.list
-echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly $(lsb_release -cs) main" > /etc/apt/sources.list.d/gazebo-nightly.list
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
-apt-get update
-apt-get install -y libignition-transport7-dev
-```
-
-
-## Install Go
-
-Go version 1.15 or above (NOTE: we are currently using 1.15.2)
--    Follow instructions from: https://golang.org/doc/install#install
-- `curl -O https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz && tar -C /usr/local -xzf go1.15.2.linux-amd64.tar.gz`
-
-
-1. Make a workspace (if needed), for example:
-
-```
-mkdir -p ~/go_ws
-```
-
-1. Download server code into new directories in the workspace:
-
-```
-hg clone https://github.com/gazebo-web/cloudsim ~/go_ws/src/github.com/gazebo-web/cloudsim
-```
-
-1. Set necessary environment variable (needs to be set every time the environment is built)
-
-```
-export GOPATH=~/go_ws
-```
-
-
-## Install `dep` tool (we are currently using v0.4.1) to manage Go dependencies (in vendor/ folder)
-
-Create a bin directory
-
-```
-mkdir ~/go_ws/bin
-```
-
-Move to the workspace's root
-
-```
-cd ~/go_ws
-```
-
-Install dep tool
-
-```
-export DEP_RELEASE_TAG=v0.4.1
-curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-```
+  <img src="./assets/logo.png" width="200" alt="Gazebo" />
+  <h1>Gazebo Cloudsim</h1>
+  <p>Cloudsim is an open source library to run robotic applications in the cloud. It currently supports deployments using Kubernetes and AWS resources, but support for other providers can be implemented.</p>
 
+[![GitHub release](https://img.shields.io/github/release/gazebo-web/cloudsim?include_prereleases=&sort=semver&color=blue)](https://github.com/gazebo-web/cloudsim/releases/)
+[![License](https://img.shields.io/badge/license-Apache-blue)](#license)
+[![issues - cloudsim](https://img.shields.io/github/issues/gazebo-web/cloudsim)](https://github.com/gazebo-web/cloudsim/issues)
+[![Go Report Card](https://goreportcard.com/badge/github.com/gazebo-web/cloudsim)](https://goreportcard.com/report/github.com/gazebo-web/cloudsim)
```

Comment:
This is showing up as a broken link to me.

Reply:
Works for me, care to try again? https://github.com/gazebo-web/cloudsim/blob/a00e77d0acfa864360f588593d46c01e01a97034/README.md

Reply:
Working now. Was probably an issue on my end.